src/app/search/search.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container-bg">
+<div class="t-100">
+    <h3>Your search parameters: {{this.query}}</h3>
+    <form class="d-flex">
+        <input [(ngModel)]='searchDetails' name="search" class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+        <button (click)="onSearch()" class="btn btn-outline-success" type="submit">Search</button>
+      </form>
+    <hr>
+    <div class="text-center">   
+        <h2>Products</h2>
+    </div>
+    <div class="wrapper-products t-100">
+        <div *ngFor="let product of products" (click)='toProductDetail(product.id)' data-id="product.id">
+              <div class="card m-3" style="width: 16rem;">
```

Comment:
Again inline styles are a no-go for me. Also Depending on the resolution width the cards look weird and don't take up the whole width.
My proposal would be using flex-box here for styling

Reply:
I've removed inline styles and added 'justify-content' parameter to flex-box